client/battle/CreatureAnimation.cpp

```diff
@@ -115,12 +115,12 @@ float AnimationControls::getCreatureAnimationSpeed(const CCreature * creature, c
 
 float AnimationControls::getProjectileSpeed()
 {
-	return static_cast<float>(settings["battle"]["animationSpeed"].Float() * 100);
+	return static_cast<float>(settings["battle"]["animationSpeed"].Float() * 4000);

```

Comment:
multiplier of projectile speed compared to multiplier of catapult speed has different ratio now, is it intended?

Reply:
Hmm... Not deliberate - I picked sensible values for both projectile types, without taking into account previous ratio.
I doubt that either old or new values match H3 settings, so don't think that there is any *need* to maintain same ratio. 
Proper solution would be to actually compare these values with H3, but same can be said about pretty much any number in this class.